Translation to Portuguese from 'Remember: An interactive comic on Spaced Repetition' by Nicky Case

fititnt opened the pull request:
refs: https://github.com/ncase/remember/issues/5

---
- Diff: https://github.com/fititnt/remember/pull/1/files?utf8=%E2%9C%93&diff=split
- Preview: https://fititnt.github.io/remember/pt.html

---
NOTICE 1: when possible, translations that could make harder to understand from other Portuguese dialects should be minimized. This could make more impact on jokes. **Native Portuguese speakers from other regions of the world than Brazil are very welcome to give feedback.**


**NOTICE 2: the translation preview to Portuguese have some text that go beyond the margins of the boxes.** This can be solved both by changing the default font size or for each individual box. At this moment (2018-10-29 13:12 BRT) the focus are more on get the translation OK than the font sizes, in special because after some text be added, we can define a default ( in the English version is `body {font-size: 25px;}`) that could give less work for change each individual box.

---
Terms that needs some extra attention on translation (good candidates for focused discussion):

- Super important ones:
  - Spaced Repetition (_Repetição Espaçada_)
  - Forgetting Curve (_Curva **do** Esquecimento_)
  - How To Remember Anything Forever-ish (_Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre_)
- Others (that needs requires check the consistency)
  - recall (_rememoração_)
  - increasing gaps / gaps increase
  - increasing gaps, decreasing gaps, random gaps
  - cramming (_Não foi traduzido para português_)
  - memory decay, decay

fititnt committed:
translation-pt: <html lang='pt'>

fititnt committed:
translation-pt: meta stuff

fititnt commented:
Previews here https://fititnt.github.io/remember/pt.html

fititnt committed:
translation-pt: 'Spaced Repetition' as 'Repetição Espaçada' (Note: PT still does not have even a draft for this term yet)

fititnt committed:
translation-pt: added data to translations.txt & some more pt translations to main file

fititnt committed:
translation-pt: 'Forgetting curve' as 'Curva do Esquecimento' (Note: PT still does not have even a draft for this term yet)

Lawendt commented on pt.html:
```diff
@@ -28,32 +28,32 @@
 
 	<!-- Meta Stuff -->
-	<title>How To Remember Anything Forever-ish</title>
+	<title>Como Se Lembrar De Qualquer coisa Para Quase Sempre</title>
```
```suggestion
	<title>Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre</title>
```

Acredito que `Coisa` deveria estar com maiúsculo também e acho que o quase entre parenteses faz mais sentido para passar a mesma ideia do `Forever-ish`

fititnt replied:
Sim, vou por o "Coisa" em maiúsculo. Porém o título em si ainda está em aberto (ou seja, da ate pra pensar algo diferente além de por coisa como fiz).

Até no repositório original você vai ver eu perguntando o que as outras pessoas estão usando para traduzir.

Lawendt commented on pt.html:
```diff
@@ -28,32 +28,32 @@
 	<!-- Sharing Card Stuff -->
-	<meta itemprop="name" content="How To Remember Anything Forever-ish"> <!-- TRANSLATE "content" -->
-	<meta itemprop="description" content="an interactive comic on the art & science of memory"> <!-- TRANSLATE "content" -->
+	<meta itemprop="name" content="Como Se Lembrar De Qualquer coisa Para Quase Sempre"> <!-- TRANSLATE "content" -->
```
```suggestion
	<meta itemprop="name" content="Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre"> <!-- TRANSLATE "content" -->
```

Lawendt commented on pt.html:
```diff
@@ -28,32 +28,32 @@
 
-	<meta name="twitter:title" content="How To Remember Anything Forever-ish"> <!-- TRANSLATE "content" -->
-	<meta name="twitter:description" content="an interactive comic on the art & science of memory"> <!-- TRANSLATE "content" -->
+	<meta name="twitter:title" content="Como Se Lembrar De Qualquer coisa Para Quase Sempre"> <!-- TRANSLATE "content" -->
```
```suggestion
	<meta name="twitter:title" content="Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre"> <!-- TRANSLATE "content" -->
```

Lawendt commented on pt.html:
```diff
@@ -28,32 +28,32 @@
 
-	<meta property="og:title" content="How To Remember Anything Forever-ish"> <!-- TRANSLATE "content" -->
-	<meta property="og:description" content="an interactive comic on the art & science of memory"> <!-- TRANSLATE "content" -->
+	<meta property="og:title" content="Como Se Lembrar De Qualquer coisa Para Quase Sempre"> <!-- TRANSLATE "content" -->
```
```suggestion
	<meta property="og:title" content="Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre"> <!-- TRANSLATE "content" -->
```

Lawendt commented on pt.html:
```diff
@@ -77,13 +77,13 @@
-			<div>
-			HOW TO REMEMBER ANYTHING FOREVER-ISH
+			<div style="font-size: 0.75em">
+			COMO SE LEMBRAR DE QUALQUER COISA PARA QUASE SEMPRE
```
```suggestion
			COMO SE LEMBRAR DE QUALQUER COISA PARA (QUASE) SEMPRE
```

Lawendt commented on pt.html:
```diff
@@ -122,25 +122,25 @@
 <panel w=600 h=60>
 <words w=600 x=-15 no-bg>
-	So, how's Memory and Inspiration doing in schools?
+	Então, como Memória e Inspiração é aplicada nas escolas?
```
```suggestion
	Então, como Memória e Inspiração são aplicadas nas escolas?
```
Não seria plural aqui?

fititnt replied:
Eu não tenho certeza absoluta, então vou aceitar

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
 <pic src="pics/intro0.png" sx=400 sy=250></pic>
 <words x=155 y=20 w=90 no-bg>
-	Yeah.
+	É assim.
```
Talvez possa traduzir para algo mais com a expressão `Yeah`, talvez: `Pois é`.

fititnt replied:
Ótima sugestão! E olha que parei pra pensar bastante nessa parte aqui. No mínimo fica melhor do que o atual

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
 <words x=196 y=65 w=270 no-bg>
-	Not only are common practices like lectures, cramming, and re-reading <i>boring</i>,
-	science has shown they <i>don't even work well</i>.*
+	Não apenas as práticas comuns, como palestras, <i><abbr title="Ato de decorar quantidades massivas no dia anterior de um teste" lang="en">cramming</abbr></i>
```
```suggestion
	Não apenas práticas comuns como palestras, <i><abbr title="Ato de decorar quantidades massivas no dia anterior de um teste" lang="en">cramming</abbr></i>
```

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
-	science has shown they <i>don't even work well</i>.*
+	Não apenas as práticas comuns, como palestras, <i><abbr title="Ato de decorar quantidades massivas no dia anterior de um teste" lang="en">cramming</abbr></i>
+	e releituras são <i>chatas</i>,
+	a ciência tem mostrado que <i>nem mesmo funcionam bem</i>.*
```
```suggestion
	a ciência mostra que <i>nem funcionam bem</i>.*
```
Tentando tirar alguns caracteres para ficar dentro do balão

Edit: problema do balão não existe mais.

Lawendt replied:
```suggestion
	a ciência tem mostrado que <i>nem sequer funcionam bem</i>.*
```

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
 <words x=95 y=30 w=340 no-bg>
-	But what if I said there's a way to learn
-	that's evidence-based <i>and</i> fun?
+	Mas e se eu disser que há uma maneira de aprender que é baseada em evidências <i>e</i> divertida?
```
Essa não encaixa no balão.

Só pensei em mudar para:

`Mas e se eu disser que há uma maneira que funciona?`

fititnt replied:
Por enquanto, acho que vale a pena a gente se focar mais na tradução dos textos para português, e depois (se reeeealmente for necessário) alterar frases para forçar se ajustar com o que está no original.

Porém todos as alterações com traduções em si são extremamente interessantes

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
 <words x=164 y=141 w=300 no-bg>
-	What if I said there's a memory card game you can play, for 20 minutes a day,
-	to store <i>anything you choose</i> into long-term memory, <i>forever?*</i>
+	E se eu dissesse que há um jogo de cartão de memória que você pode jogar, por 20 minutos por dia,
```
Não encaixa no balão.

Talvez tirar `de cartão de memória`

fititnt replied:
Idem https://github.com/fititnt/remember/pull/1#discussion_r228934378

Lawendt commented on pt.html:
```diff
@@ -161,36 +161,42 @@
-	What if I said there's a memory card game you can play, for 20 minutes a day,
-	to store <i>anything you choose</i> into long-term memory, <i>forever?*</i>
+	E se eu dissesse que há um jogo de cartão de memória que você pode jogar, por 20 minutos por dia,
+	para armazenar <i>qualquer coisa que escolher</i> na memória de longo prazo, para <i>sempre?*</i>
```
Não encaixa no balão.

Talvez tirar `que escolher`

fititnt replied:
Idem https://github.com/fititnt/remember/pull/1#discussion_r228934378

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
-	And it's <i>awesome</i>.
-	I started using Spaced Repetition earlier this year to learn French.
+	E é <i>incrível</i>.
+	Comecei a usar o Repetição Espaçada no início deste ano para aprender francês.
```
Não encaixa no balão.

Acho que seria **a** Repetição Espaçada. 

Acho que poderia tirar o `no início` para diminuir a frase.

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
-	In two <i>months</i>, I learnt more words than I did in two <i>years</i>
-	of high school French classes.
+	Em dois <i>meses</i> eu aprendi mais palavras do que em dois <i>anos</i>
+	de aulas de francês no ensino médio.
```
Não encaixa no balão.

Acho que dá para tirar `de aulas` que continua o mesmo sentido
```suggestion
	de francês no ensino médio.
```

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 <pic src="pics/intro0.png" sx=950 sy=450></pic>
 <words x=10 y=10 w=400 h=60>
-	Since then, I've used Spaced Repetition to remember all sorts of things...
+	Desde então, passei a usar a Repetição Espaçada para lembrar de todo tipo de coisa...
```
Não encaixa no balão.

Talvez mudar para: `qualquer coisa`, `várias coisas`

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 </words>
 <words x=60 y=300 w=400 h=60>
-	...and this lil' memory card game became a core part of my <i>life</i>.
+	...e este pequeno jogo de cartas de memória tornou-se uma parte essencial da minha <i>vida</i>.
```
```suggestion
	...e este joguinho de cartas de memória tornou-se uma parte essencial da minha <i>vida</i>.
```

fititnt replied:
Realmente, ficou melhor. Mais informal, porém ainda não no nível de ficar complicado para os demais dialetos de português. Já aplicando isso no proximo commit.

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 <words x=10 y=220 w=500 h=90>
-	You test yourself on a fact repeatedly, spacing out your repetitions over time.
-	(But won't this take forever? Ah, as we'll see later, there's a trick...)
+	Você se testa em um fato repetidamente, espaçando suas repetições ao longo do tempo.
```
Não encaixa no balão.

Talvez:
```suggestion
	Você se testa em um fato repetidamente, espaçando ao longo do tempo.
```

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 <pic src="pics/intro0.png" sx=0 sy=1200></pic>
 <words x=10 y=20 w=290 no-bg>
-	Spaced Repetition is free, evidence-based, and so simple you can do it with a <i>shoebox</i>.
+	Repetição Espaçada é gratuita, baseada em evidências, e tão simples que você pode fazê-lo com uma <i>caixa de sapatos</i>.
```
Não encaixa no balão.

```suggestion
	É gratuita, baseada em evidências, e tão simples que dá para fazer com uma <i>caixa de sapatos</i>.
```

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 </words>
 <words x=292 y=303 w=80 no-bg class="comic_text" style="text-align:left; font-size:20px">
-	teacher's coming
+	professor está vindo
```
Acho que dá para mudar aqui e deixar mais informal

```suggestion
	professor tá vindo
```

ou até

```suggestion
	sor tá vindo
```

😛

fititnt replied:
Esse aqui dependendo deixa marcado com estrelinha pra discutir mais tarde.

Ate pensando em revisar isso com contatos meus que são de outros países que falam português, mas o dialeto deles não é pt-BR

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
 <words x=10 y=20 w=330 no-bg>
-	Well, the catch is that making <i>any</i> new habit is hard
-	– especially a weird habit like Spaced Repetition.
+	Bem, a pegadinha é que fazer <i>qualquer</i> novo hábito é difícil
```
Acho que seria `criar` e não `fazer` 🤔 

E como não encaixa no balão acho que poderia tirar `a pegadinha é que`

Lawendt commented on pt.html:
```diff
@@ -201,96 +207,97 @@
-	<b>That is: you'll use Spaced Repetition to learn about Spaced Repetition.</b>
+	Além disso, ao longo deste quadrinho,
+	você pode testar o que aprendeu
+	em intervalos regulares.
```
Os intervalos não são `regulares`!

```suggestion
	em intervalos espaçados.
```

Lawendt commented on pt.html:
```diff
@@ -302,44 +309,49 @@
-	Still, isn't "rote memorization" bad?
-	Can't we look everything up these days?
-	Shouldn't we learn creativity &amp; critical thinking instead?
+	Opa, mas o "habito de memorizar" não é ruim?
```
Acho que habito de memorizar não condiz com o termo de rote memorization né?
```suggestion
	Opa, mas "memorização mecânica" não é ruim?
```

fititnt replied:
Hummmmmmm, pelo visto você diz isso aqui https://en.wikipedia.org/wiki/Rote_learning. Mais tarde quando acordar respondo com calma.

Pior que vários destes termos não tem nem um draft na wikipedia. Até comentei lá no post original com o tradudor pra alemão de que talvez valesse a pena rascunhas as respectivas páginas em breve.

Lawendt commented on pt.html:
```diff
@@ -302,44 +309,49 @@
 <pic src="pics/intro0.png" sx=550 sy=1600></pic>
 <words x=20 y=20 w=330 h=30>
-	...to mother your own, inner Muse.
+	...para cuidar da sua própria Musa interior.
```
talvez tirar `própria` para tentar encaixar?

fititnt replied:
Os tamanhos de fonte possivelmente vou deixar para ajustar depois de traduzir mais uma boa parte. E vai ter muito, mas muito ajuste.

(depois volto pra responder os outros pontos)

fititnt committed:
translation-pt: 'sweet spot' as 'ponto ideal'; more translations

Lawendt commented on pt.html:
```diff
@@ -2372,13 +2370,13 @@
 	<div class="fcard_bg" src="pics/fcards0.png" sx=0 sy=0></div>
 	<div class="fcard_center" style="height:2.5em">
-	...with <i>increasing</i> gaps!
+	...com laculas <i>crescentes</i>!
```
typo!

´lacunas´ ao invés de ´laculas´

fititnt committed:
translation-pt: review by @Lawendt, 10 changes:

- Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre
+ Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre

- COMO SE LEMBRAR DE QUALQUER COISA PARA QUASE SEMPRE
+ COMO SE LEMBRAR DE QUALQUER COISA PARA (QUASE) SEMPRE

- Então, como Memória e Inspiração é aplicada nas escolas?
+ Então, como Memória e Inspiração são aplicadas nas escolas?

- É assim.
+ Pois é.

- Não apenas as práticas comuns, como palestras,
+ Não apenas práticas comuns como palestras,

- ...e este pequeno jogo
+ ...e este joguinho

- Bem, a pegadinha é que fazer
+ Bem, a pegadinha é que criar

- em intervalos regulares.
+ em intervalos espaçados.

- Opa, mas o "habito de memorizar" não é ruim?
+ Opa, mas "memorização mecânica" não é ruim?

- ...com laculas <i>crescentes</i>
+ ...com lacunas <i>crescentes</i>

fititnt commented:
Olha que lindo essa interface do GitHub. Ela até mesmo mostra "outdated" nas sugestões de alteração que tiveram as linhas alteradas.

Eu devo ficar traduzindo pelo menos umas duas horas hoje, porém as sugestões que forem feitas vou procurar fazer em commits específicos pra elas (ou no caso de ser a mesma pessoa) agrupar por pessoa.

fititnt committed:
translation-pt: hotfix, CSS 'body {font-size: 20px}' as default to improve legibility of the preview page

Lawendt commented:
Sobre `recall`. Eu achei que ficou bem feio a tradução para `rememoração`. Eu acho que `recordar` traduz muito melhor e é muito mais usado. O que tu acha?

Lawendt commented on pt.html:
```diff
@@ -666,7 +688,7 @@
 <words w=600 x=-15 no-bg>
 	(To prove this isn't a fluke,
```
```suggestion
	(Pra provar que isso não é por acaso,
```

Lawendt commented on pt.html:
```diff
@@ -666,7 +688,7 @@
 <words w=600 x=-15 no-bg>
 	(To prove this isn't a fluke,
 	here’s a sim where you can change
```
```suggestion
	Aqui está um simulador em que você pode mudar
```

Lawendt commented on pt.html:
```diff
@@ -666,7 +688,7 @@
 	(To prove this isn't a fluke,
 	here’s a sim where you can change
-	the initial memory decay &amp; sweet spot.
+	the initial memory decay &amp; ponto ideal.
```
```suggestion
	a taxa inicial de esquecimento &amp; ponto ideal.
```
ou 
```suggestion
	o decaimento de memória inicial &amp; ponto ideal.
```

Lawendt commented on pt.html:
```diff
@@ -666,7 +688,7 @@
 	here’s a sim where you can change
-	the initial memory decay &amp; sweet spot.
+	the initial memory decay &amp; ponto ideal.
 	Note how, in all but the extreme cases,
```
```suggestion
	Note como, em todos casos não extremos,
```

Lawendt commented on pt.html:
```diff
@@ -666,7 +688,7 @@
-	the initial memory decay &amp; sweet spot.
+	the initial memory decay &amp; ponto ideal.
 	Note how, in all but the extreme cases,
 	the best schedule is still “increasing gaps”!)
```
```suggestion
	O melhor cronograma ainda é "lacunas incrementais"!)
```

Lawendt commented on pt.html:
```diff
@@ -680,7 +702,7 @@
 <pic src="pics/sci2.png" sx=0 sy=450></pic>
 <words x=10 y=10 w=300>
 	Why <i>must</i> the gaps increase?
```
```suggestion
	Por que as lacunas <i>devem</i> aumentar?
```

Lawendt commented on pt.html:
```diff
@@ -680,7 +702,7 @@
 <words x=10 y=10 w=300>
 	Why <i>must</i> the gaps increase?
-	Because: each time you do a recall at the sweet spot of forgetting,
+	Because: each time you do a recall at the ponto ideal of forgetting,
```
```suggestion
	Porque: cada vezes que você relembra no ponto ideal do esquecimento,
```

Lawendt commented on pt.html:
```diff
@@ -680,7 +702,7 @@
 	Why <i>must</i> the gaps increase?
-	Because: each time you do a recall at the sweet spot of forgetting,
+	Because: each time you do a recall at the ponto ideal of forgetting,
 	the memory’s decay slows down...
```
```suggestion
	a taxa de esquecimento fica mais devagar...
```

Lawendt commented on pt.html:
```diff
@@ -696,7 +718,7 @@
 </words>
 <words x=10 y=400 w=300>
 	...meaning it’ll take <i>longer</i>
```
```suggestion
	...significando que tomara <i>mais tempo</i>
```

Lawendt commented on pt.html:
```diff
@@ -696,7 +718,7 @@
 <words x=10 y=400 w=300>
 	...meaning it’ll take <i>longer</i>
-	to hit the sweet spot next time!
+	to hit the ponto ideal next time!
```
```suggestion
	para atingir o ponto ideal na próxima vez!
```

Lawendt commented on pt.html:
```diff
@@ -201,96 +218,97 @@
-	And it's <i>awesome</i>.
-	I started using Spaced Repetition earlier this year to learn French.
+	E é <i>incrível</i>.
+	Comecei a usar o Repetição Espaçada no início deste ano para aprender francês.
```
```suggestion
	Comecei a usar a Repetição Espaçada no início deste ano para aprender francês.
```

Lawendt commented on pt.html:
```diff
@@ -381,59 +404,59 @@
 </words>
 <words x=193 y=169 w=70 no-bg class="comic_text" style="color:rgba(0,0,0,0.35)">
-	plz...
+	favor...
```
No original é `plz` talvez fosse bom traduzir como abreviação também

```suggestion
	pfvr...
```

fititnt replied:
Esse é um daqueles pontos que não estou certo de mudar ou não (também pense em deixar resumido, tipo "plz" em vez pfvr de inicio). Por isso que coloque as duas chamadas seguintes foram também "me" e "matar" para ficar de acordo.

Porém, esse ponto junto com outro antes que não atualizei, creio que pode valer a pena pelo menos mais uma pessoa pra confrmar ou reconfirmar, e talvez e talvez revisar com alguem que fale portugues não do Brasil. Ou mesmo no https://portuguese.stackexchange.com (assim como pretendo fazer com alguns outros termos mais oficiais aqui).

Escrevi tudo isso porque esse é um segundo ponto que precisa ser revisto, não que não vai ser aplicado exatamente como falou.

Lawendt commented on pt.html:
```diff
@@ -381,59 +404,59 @@
 </words>
 <words x=348 y=234 w=70 no-bg class="comic_text" style="color:rgba(0,0,0,0.35)">
-	me...
+	matar...
```
```suggestion
	mate...
```

fititnt replied:
refs https://github.com/fititnt/remember/pull/1#discussion_r229550300

Lawendt commented:
Será que `memory decay` -> `esquecimento`

`rate of memory decay` ficaria `taxa de esquecimento`

fititnt commented:
> Sobre `recall`. Eu achei que ficou bem feio a tradução para `rememoração`. Eu acho que `recordar` traduz muito melhor e é muito mais usado. O que tu acha?

**ESSE** é um ponto em especial que vale a pena ver (talvez como alguns outros) em um lugar como https://portuguese.stackexchange.com/. E dai aplicar o mesmo termo em todo texto (ou quase todo texto).

Motivação: rememorar teoricamente é um termo mais exato (e, mesmo sem ter ouvido antes, da pra supor o que ele seria) porém recordar é outro termo mais comum de ouvir no dia a dia.

fititnt commented:
Informei/solicitei no repositório oficial também encaminhamento para que o vídeo que é anexado possa de alguma forma ser traduzido.

Link de referência do issue: https://github.com/ncase/remember/issues/8

fititnt commented:
**StackExchange: Portuguese Language: Termo técnico “Forgetting Curve” em português**

https://portuguese.stackexchange.com/questions/5600/termo-t%C3%A9cnico-forgetting-curve-em-portugu%C3%AAs

---

O SE PT tem uma limitação de que (ao menos um usuário com 101 de pontuação) precisar ter ao menos 40 min para repostar outra pergunta.

Outro comentário em off: talvez por mais 1~3 dias eu ainda não pare para fazer uma leva grande de traduções (no caso, ficar umas 6h+ focado e chegar próximo de uma tradução quase final). De qualquer forma, os encaminhamentos (como questões extras, ver questão de potencialmente ver vídeo de youtube também, etc) acho que já consigo ir fazendo.

Mesmo considerando as outras traduções que começaram até um dia antes, a para português não está necessariamente mais atrasada, então ainda está ok.

fititnt committed:
translation-pt: review + translation by @Lawendt, part 2

fititnt commented:
Pessoal que estiver lendo isso, tivemos resposta no stackexchange das três perguntas nos ultimos dias. Talvez eu aguarde mais um pouco antes de aprová-las lá e aqui, mas já deixo como referência para (re-)discussão

- https://portuguese.stackexchange.com/questions/5599/termo-t%C3%A9cnico-spaced-repetition-em-portugu%C3%AAs
- https://portuguese.stackexchange.com/questions/5600/termo-t%C3%A9cnico-forgetting-curve-em-portugu%C3%AAs
- https://portuguese.stackexchange.com/questions/5610/termo-equivalente-a-cramming-no-contexto-educacional

fititnt committed:
translation-pt: finished capter 'THE SCIENCE of SPACED REPETITION'

fititnt committed:
translation-pt: started chapter 2, 'THE ART OF SPACED REPETITION'

fititnt commented on pt.html:
```diff
@@ -831,58 +835,67 @@
 <words x=30 y=300 w=430 h=60>
-	First, divide your box up into seven "Levels".
-	(You can have more or fewer if you like!)
+	Primeiro, divida sua caixa em sete "níveis".
```
Aqui deixei "níveis" em caixa baixa. Tenho que rever (e ou colocar em caixa alta como os demais ou optar por manter todas as traduções de "Levels" para "níveis"

fititnt commented:
As discussed here https://github.com/ncase/remember/issues/5#issuecomment-462931331 with @ramon-mendes, I will move the ownership of this repository for him.

Commit:
gogo

Commit:
100% translated to Portuguese

fititnt commented:
@ramon-mendes Quando preparado aí, só enviar o pull request para o repositório principal. Qualquer alteração extra melhor deixar para o pessoal fazer direto lá. Mas pelo menos a gente segue a vida.

Commit:
sizes adjustments

Commit:
gogo
